guides/doc-Provisioning_Guide/topics/Resources.adoc

```diff
@@ -54,7 +57,7 @@ You can also add custom operating systems using the following procedure:
 . From the *Root Password Hash* list, select the encoding method for the root password.
 . From the *Architectures* list, select the architectures that the operating system uses.
 . Click the *Partition table* tab and select the possible partition tables that apply to this operating system.
-. Optional: if you use non-Red{nbsp}Hat content, click the *Installation media* tab and select the installation media that apply to this operating system. For more information, see xref:Configuring_Provisioning_Resources-Creating_Installation_Media[].
+. Click the *Installation media* tab and select the installation media that apply to this operating system. When you import an operting system from Red Hat CDN, you dont need to set installation media. It can be blank. For more information, see xref:Configuring_Provisioning_Resources-Creating_Installation_Media[].
```

Comment:
This change has nothing to do with the actual purpose of this exercise. 
This seems to be a stylistic choice rather than having anything to do with the differences between Satellite and Foreman. The sentence as it was, was accurate.

Reply:
Actually, there is a reason why we try to change that and we've spent some time with this :-)

Foreman user has no Red Hat Content menu entry, so it's not optional step. It is the other way around - you want to use Installation media and if you install Katello then you don't actually set Installation Media at all (Content Source overrides that setting). Does it make sense?

Reply:
Thanks for the clarification. I get it now. 
However, I would rather this was broken into ifevals if the information needs to be different for Satellite/Foreman. For Satellite, the content could remain as is, for Foreman, can add whatever you need.  It would establish best practices to make the differences more clear.

Reply:
Meeting notes: @melcorr to articulate two sentences (upstream and downstream), @rahulbajaj0509 to incoporate it into the PR. :-)

Reply:
ifeval for Satellite has the old sentence as is: 

 Optional: if you use non-Red{nbsp}Hat content, click the *Installation media* tab and select the installation media that apply to this operating system. For more information, see xref:Configuring_Provisioning_Resources-Creating_Installation_Media[].

Ifeval for Foreman: 

Click the *Installation Media* tab and enter the information for the installation media source. For more information, see or more information, see xref:Configuring_Provisioning_Resources-Creating_Installation_Media[].

In my opinion, there is no need to mention Red Hat CDN here. What do you think @lzap 

The topic on installation media will need to be edited also with ifevals because I was asked to edit it recently to make it *really obvious* that you only need to do this for third party, non Red Hat content, because Red Hat customers had been confused .